shepherd/LCM.py

```diff
@@ -32,13 +29,10 @@ def run():
     rec_thread.run = run
     rec_thread.start()
 
-def lcm_send(target_channel, header, *args):
+def lcm_send(target_channel, header, dic):
```

Comment:
might be a bit confusing, but api should support not having the `dic` argument filled in. in that case it should just send an empty dictionary